### PR TITLE
Fix : #4793  

### LINKED ISSUE #4793: Talkback is reading close icon as navigator button in Hints page
**Describe the bug**
Talkback is reading close icon as navigator icon in Hints page

**To Reproduce**
Enable Talkback
1. Go to 'Placevalues' - What are the place values
2. Click on Continue till you reach the question 'how many place values does your score have, Jaime'
3. Wait till you get a hint

**Expected behavior**
Talkback should read close icon instead of navigator button

**Demonstration**
https://user-images.githubusercontent.com/101634267/208364800-d327449c-e829-4a1d-9d1b-8c99fe0a6134.mp4

**Environment**
Device/emulator being used: Infinix smart 5A
Android or SDK version (e.g. Android 5 or SDK 21): Android version 12
App version (you can get this through system app settings or via the admin controls menu in-app): 0.10-beta-5e64fae55e

### PR DESCRIPTION
**Pull request by prafulbbandre**
## Explanation
Fixes #4793 

Note: The exploration player page close button was also saying navigate up so i changed it to close.
## Essential Checklist

- [X] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [X] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [X] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

https://github.com/oppia/oppia-android/assets/109658545/e542f6fc-02fc-43de-a413-93fe1e49bc04

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by prafulbbandre** [comment]
@adhiamboperes PTAL
Thanks a lot for help.

File: app/src/main/res/layout/resume_lesson_activity.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
         android:background="@color/component_color_shared_activity_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationContentDescription="@string/navigate_up"
+        app:navigationContentDescription="@string/hints_andSolution_close_icon_description"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/resume_lesson_activity.xml]: I think this is out of scope for this PR. Please remove this change.

This is the resume lesson screen:
Screenshot_1689663974

File: app/src/main/res/layout/exploration_activity.xml [hunk 2]
```diff
@@ -31,7 +31,8 @@
         android:background="@color/component_color_shared_secondary_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationIcon="@drawable/ic_close_white_24dp">
+        app:navigationIcon="@drawable/ic_close_white_24dp"
+        app:navigationContentDescription="@string/hints_andSolution_close_icon_description">
```
Comment (adhiamboperes) [inline app/src/main/res/layout/exploration_activity.xml]: If you want it to read as close, that's fine, but reusing the string that is named after a different screen is not advisable. You should create a new string for it named after the file. Please open `strings.xml` and search for "close" so that you can see examples.

File: app/src/main/res/layout/resume_lesson_activity.xml (hunk 1 above)
Comment (adhiamboperes) [reply app/src/main/res/layout/resume_lesson_activity.xml]: This is also resulting in test failure:

```
1) testResumeLessonActivity_onToolbarClosePressed_closesResumeExplorationActivity(org.oppia.android.app.resumelesson.ResumeLessonActivityTest)
androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with content description from resource id: <2131821006>[nav_app_bar_navigate_up_description] value: Navigate up
```

File: app/src/main/res/layout/exploration_activity.xml (hunk 2 above)
Comment (adhiamboperes) [reply app/src/main/res/layout/exploration_activity.xml]: Also note that we add tests for these content descriptions, so changing it here will definitely cause failures in ExplorationActivityTest. Please see [wiki](https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results) for reference on intepreting CI failures, in order to know which tests you need to update.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @prafulbbandre! 

I have left some comments inline. Please also update the PR title with issue title.

File: app/src/main/res/layout/resume_lesson_activity.xml (hunk 1 above)
Comment (prafulbbandre) [reply app/src/main/res/layout/resume_lesson_activity.xml]: I dont know why but in my app the back button is replaced with the close icon as it is in the hint.
Let me just confirm.

Comment (prafulbbandre) [reply app/src/main/res/layout/resume_lesson_activity.xml]: Done.

File: app/src/main/res/layout/exploration_activity.xml (hunk 2 above)
Comment (prafulbbandre) [reply app/src/main/res/layout/exploration_activity.xml]: Ok @adhiamboperes,
For this Pr i will just make the necessary chages asked in the issue.

**Comment by prafulbbandre** [comment]
Hi @adhiamboperes PTAL

File: app/src/main/res/layout/resume_lesson_activity.xml [hunk 3]
```diff
@@ -21,7 +21,7 @@
         android:background="@color/component_color_shared_activity_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationContentDescription="@string/hints_andSolution_close_icon_description"
+        app:navigationContentDescription="@string/Navigate up"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/resume_lesson_activity.xml]: This string format is incorrect, and the app would not have built successfully like this. Generally, please test your changes manually before pushing the code.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Hey @prafulbbandre, thank you, I have reviewed your changes and left a comment in-line.

Please also: 
1. Review the wiki section on [submitting a PR](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#step-3-address-review-comments-until-all-reviewers-give-lgtm). Let the reviewer resolve the comment threads themselves, unless otherwise specified.
2. Please update the video in the description once you confirm your latest changes.

File: app/src/main/res/layout/resume_lesson_activity.xml (hunk 3 above)
Comment (prafulbbandre) [reply app/src/main/res/layout/resume_lesson_activity.xml]: Done

**Review by adhiamboperes — APPROVED** [review]

**Comment by adhiamboperes** [comment]
Thanks @prafulbbandre.
